src/Pim/Bundle/CatalogBundle/tests/integration/PQB/Filter/Options/LocalizableFilterIntegration.php

```diff
@@ -21,6 +21,8 @@ protected function setUp()
         parent::setUp();
 
         if (1 === self::$count || $this->getConfiguration()->isDatabasePurgedForEachTest()) {
+            $this->resetIndex();
```

Comment:
imho, we should reset index on https://github.com/jjanvier/pim-community-dev/blob/TIP-613-unified/tests/TestCase.php#L41 to avoid to call this method on each integration test